web-app/src/main/webapp/css/style.css

```diff
@@ -1,9 +1,26 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#map {
+  /* placeholder until we add a map */
+  background-color: #add8e6;
+  height: 100vh;
+  position: absolute;
+  width: 100%;
+  z-index: 1;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#addressForm {
+  /* overwrite skeleton margins on form */
+  margin-top: 50px;
+  margin-bottom: 0;
+  position: absolute;
+  width: 100%;
+  z-index: 2;
+}
+
+#searchInput {
+  background: #fffcfc;
+  border: 2px solid #9e9e9e;
+  border-radius: 6px;
+  box-sizing: border-box;
+  font-size: 30px;
+  height: 5vh;
```

Comment:
nit: maybe use "em" units since the height should probably be relative to font-size and not the height of the screen?